README.md

```diff
@@ -47,33 +47,33 @@ In particular, Bounter implements three different algorithms under the hood, dep
 
 1. **[Cardinality estimation](https://en.wikipedia.org/wiki/Count-distinct_problem): "How many unique items are there?"**
 
-  ```python
-  from bounter import bounter
+```python
+from bounter import bounter
 
-  counts = bounter(need_counts=False)
-  counts.update(['a', 'b', 'c', 'a', 'b'])
+counts = bounter(need_counts=False)
+counts.update(['a', 'b', 'c', 'a', 'b'])
 
-  print(counts.cardinality())  # cardinality estimation
-  3
-  print(counts.total())  # efficiently accumulates counts across all items
-  5
-  ```
+print(counts.cardinality())  # cardinality estimation
```

Comment:
@isamaru Is this needed? Removing the indent changes the layout semantics: the list sections are no longer lists, incorrect vertical alignment.